Content.Server/GameObjects/Components/Damage/DamageableComponent.cs

```diff
@@ -120,6 +135,42 @@ public void TakeDamage(DamageType damageType, int amount, IEntity source = null,
                 _currentDamage[DamageType.Total] = Math.Max(0, _currentDamage[DamageType.Total] + amount);
                 UpdateForDamageType(DamageType.Total, oldTotalValue);
             }
+
+            PlayDamageSound(Owner, damageType);
+        }
+
+        
+
+        /// <summary>
+        /// Handles playing a damage sound from entity damaged.
+        /// </summary>
+        public void PlayDamageSound(IEntity entity, DamageType damageType, AudioParams? audioParams = null)
+        {
+            if(_soundArray != null)
+            {
+                if (_soundArray.TryGetValue(damageType, out var soundEffect))
+                {
+                    if(Uri.IsWellFormedUriString(soundEffect, UriKind.RelativeOrAbsolute))
```

Comment:
Using Uri feels dirty here.

Also it should use either the collection or a specific sound, not both